src/System.Management.Automation/engine/Modules/ScriptAnalysis.cs

```diff
@@ -89,19 +89,6 @@ internal static ScriptAnalysis Analyze(string path, ExecutionContext context)
             return result;
         }
 
-        internal static string ReadScript(string path)
```

Comment:
@MartinGC94 Do you think it is good to remove the method? Won't this code be less readable?

Reply:
I think it's perfectly fine. If anything it's arguably easier to understand when it's just `File.ReadAllText` because you don't have to check if `ReadScript` does anything weird.